PROJECT-BRIEF.md

```diff
@@ -1,59 +1,186 @@
+| [Communication for Devs Presentation](https://youtu.be/LVjG3qxtQX0) ([slides](https://docs.google.com/presentation/d/14kJTPHCVHsEmzqOLTqKCgL3QrwR7_BnIpN9oGJraT4c/edit#slide=id.p)) |
+| [Pair Programming Presentation](https://youtu.be/yFVJ2gph_Uc) (slides)                                                                                                              |
+| Agile Presentation (slides only)                                                                                                                                                    |
+
+| 👋 Sync meetings | Agenda                                                                                                                                                                                                                                   |
+| ---------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Weekly Sync      | - Mentors pair live through issue #1 so Collabies can get a demostration of the full development cycle at Collab Lab and we can ensure everyone has Firebase auth for their app <br> - Collabies will be presented with issues #2 and #3 |
+| Pair Programming | Schedule a pairing session(s) with your partner early in the week to complete the development work required to meet the acceptance criteria of your Github issue.                                                                        |
+| Optional         | Office Hours                                                                                                                                                                                                                             |
+
+### Week 2, {DATES_WEEK_2}
+
+| 🤝 Developer pairings | 🖥 Assigned Github issue                                                                                              |
+| --------------------- | --------------------------------------------------------------------------------------------------------------------- |
+| {DEV_2} & {DEV_3}     | [4. As a user, I want to set up a new shopping list so I can start tracking purchased items.]({REPO_URL}/issues/4)    |
+| {DEV_1} & {DEV_4}     | [5. As a user, I want to add a new item to my shopping list so I can start recording purchases.]({REPO_URL}/issues/5) |
+
+| 📖 Learning modules                                                                                                                                           | Description |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| [Accessibility](https://youtu.be/xrP2DIoZRnQ) ([slides](https://docs.google.com/presentation/d/1ux4HXGj3XAGZRD7qvOpjl36FYbTcUYGaaBHywc95taY/edit#slide=id.p)) |
```

Comment:
**Possible Description:** In this module, you'll learn about inclusive practices to ensure there are no barriers to prevent users from interacting with your application. 

_Bulleted description, if needed:_
You'll learn:

- what accessibility is
- why it is important
- best practices for writing accessible code